bower.json

```diff
@@ -14,7 +14,7 @@
   "devDependencies": {
     "uglify-js": "~2.2",
     "jison": ">=0.2.0",
-    "highlight.js": "~8.0.0",
+    "highlight.js": "~9.6.0",
```

Comment:
The `bower.json` is (was) for people using Bower as part of their compilation steps. Not sure it's needed anymore.